package.json

```diff
@@ -24,6 +25,13 @@
   "bugs": {
     "url": "https://github.com/meetalva/alva/issues"
   },
+  "lint-staged": {
+    "*.{ts,tsx}": [
+      "prettier --write",
+      "tslint -c tsconfig.json --fix",
+      "git add"
```

Comment:
What happens with partial commits? Would they not be added completely?

Reply:
That is true for commits that contain file chunks, don't know if this is common practice in the team?

Reply:
Good question. I just remembered that we had that problem before in a different project.
But i think we can merge it.

Can you just rebase this branch? I am very sure that a lot of the commits are already in the master. This way it more clear what new stuff comes in

Reply:
Done